molgenis-scripts-core/src/main/java/org/molgenis/script/core/exception/ScriptExecutionException.java

```diff
@@ -0,0 +1,38 @@
+package org.molgenis.script.core.exception;
+
+import java.text.MessageFormat;
+
+import static java.lang.String.format;
+import static org.molgenis.data.i18n.LanguageServiceHolder.getLanguageService;
+
+@SuppressWarnings("squid:MaximumInheritanceDepth")
+public class ScriptExecutionException extends ScriptException
+{
+	private static final String ERROR_CODE = "SC04";
+
+	public ScriptExecutionException(String causeMessage)
+	{
+		super(ERROR_CODE, new RuntimeException(causeMessage));
```

Comment:
Will generate a stack trace for this line, which will be confusing. Store causeMessage in field instead. (And store cause.getLocalizedMessage() in that field in the other constructor)